mis_builder/views/mis_report_instance.xml

```diff
@@ -104,6 +105,7 @@
                     <group>
                         <group>
                             <field name="report_id" string="Template" />
+                            <field name="sequence" />
```

Comment:
Is that really useful to add it in the form? Most users won't know what to do with it.

Reply:
In a scenario where the client has 50–60 reports and wants to move one of the ones at the top, it's useful to be able to change the sequence manually and assign a specific value, instead of having to drag it through all the others to find the right position. 

It's not really something we need, but it can be useful.

Reply:
I would start without and add it later if the use case really arises.